+++
title = "CWE-396: Declaration of Catch for Generic Exception\n"
description = "Catching overly broad exceptions promotes complex error handling code that is more likely to contain security vulnerabilities.\n"
weight = 396

[extra]
id = 396
name = "Declaration of Catch for Generic Exception\n"
url = "https://cwe.mitre.org/data/definitions/396.html"
class = "Base"
rust_docs_links = []
parent = "705"

[extra.vote]
"No Help, or Langs Wont Help" = false
Discouraged = false
"Discouraged via Library" = false
"Discouraged via Borrow Checker" = false
"Discouraged via Debug Mode" = false
"Discouraged via Clippy" = false
"Virtually Impossible" = true

[extra.vector]

+++

Exception handling doesn't work this way in Rust